packages/discord-bot/src/handlers/help.ts

```diff
@@ -0,0 +1,37 @@
+import { Collection, CommandInteraction, MessageEmbed } from 'discord.js';
+import { Command } from 'src/interfaces/Command';
+
+export const helpHandler = async (
+  interaction: CommandInteraction,
+  commands: Collection<string, Command>
+): Promise<void> => {
+  const cmd = interaction.options.getString('cmd');
+  if (!cmd) {
+    await interaction.editReply('Help Command');
```

Comment:
I think rather than responding 'Help Command' we should either make the 'cmd' parameter required, or improve the default message to display *all* the commands' help text.

Reply:
Actually, I think this text can be edited a bit and we could make this a tldr about how to signup and use praise".
The "Help Command" text is a placeholder for now. (Though, if needed, we can make the cmd option required)